run.sh

```diff
@@ -14,7 +14,16 @@ sudo -E python3 screen-calendar-get.py
 
 figlet Export
 # Inkscape can't export to BMP, so let's export to PNG first. 
-inkscape  screen-output-weather.svg --without-gui -e screen-output.png -w640 -h384 --export-dpi=150
+
+if [ $WAVESHARE_EPD75_VERSION = 1 ]; then 
+    WAVESHARE_WIDTH=640
+    WAVESHARE_HEIGHT=384
+else
+    WAVESHARE_WIDTH=800
+    WAVESHARE_HEIGHT=480
+fi
```

Comment:
And here we set the height width.  So the user just has to modify one line in env.sh

Reply:
yeah -- this seems reasonable.